varats/settings.py

```diff
@@ -82,8 +82,23 @@
 
 CFG['experiment'] = {
     "only_missing": {
-        "default": True,
-        "desc": "Only run missing version"
+        "default":
+        True,
+        "desc":
+        "Only run missing version [Deprecated]"
+        "This option is replaced by file_status_blacklist = [Success]"
+    },
+    "file_status_blacklist": {
+        "default": [],
```

Comment:
Do we want to have the same default behaviour as as before? If yes, you should probably do
```suggestion
        "default": [Success],
```

Also, should blocked revisions be blacklisted by default? That's why we introduced them, right?
```suggestion
        "default": [Blocked],
```

Of course, we also could do both
```suggestion
        "default": [Success, Blocked],
```